File: AmazonDemo.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class AmazonDemo {

	public static void main(String[] args) {
		System.setProperty(
			"webdriver.chrome.driver",
			"./drivers/chromedriver"
		);
		
		WebDriver driver = new ChromeDriver();
		driver.get("http://www.amazon.com/");
		
		// I am in the amazon home page
		WebElement input = driver.findElement(By.id("twotabsearchtextbox"));
		input.sendKeys("Harry Potter");
		
		WebElement searchIcon = driver.findElement(By.id("nav-search-submit-button"));
		searchIcon.click();
	}

}

File: amazon_demo.py
from selenium import webdriver

driver = webdriver.Chrome(
    executable_path='./drivers/chromedriver'
)

driver.get('http://www.amazon.com/')
# I am in the amazon home page.
input_elem = driver.find_element_by_id('twotabsearchtextbox')
input_elem.send_keys('Harry Potter')
search_icon = driver.find_element_by_id('nav-search-submit-button')
search_icon.click()